notebooks/224-stable-diffusion-text-to-image/README.md

```diff
@@ -0,0 +1,33 @@
+# Text-to-image generation with Stable Diffusion and OpenVINO
+
+Text-to-image generation is task of AI, which aim is create image from given text description.
+In this demo we use the **[Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion)** model for image generation with OpenVINO.
+
+The complete pipeline of this demo's notebook is shown below.
+
+![image2](https://raw.githubusercontent.com/patrickvonplaten/scientific_images/master/stable_diffusion.png)
+
+This is a demonstration in which the user can type the text and the pipeline will generate image which correspond to given text, step by step iteratively denoise latent image representation while being conditioned on the text embeddings provided by text encoder. 
```

Comment:
"will generate image which correspond to given text" -> "will generate an image which corresponds to given text"

Reply:
fixed